app-office/visidata/visidata-2.1.1.ebuild

```diff
@@ -53,6 +53,7 @@ python_prepare_all() {
 python_test() {
 	git init || die "Git init failed."
 	git add tests/golden/ || die "Git add failed."
+	# this test script eventually calls pytest under the hood
 	dev/test.sh || die "Tests failed."
 	rm .git -rf || die "Could not clean up git test directory."
```

Comment:
While here, could you also remove the `-f` please, and only make it `rm -r`. The `-f` makes it never fail.

Reply:
whoops too slow on commenting.
Will change it myself.